### DISCUSSION TITLE
Welcome to Batman_Robot Discussions!

### DISCUSSION THREAD

**Discussion by MrHonekawa** [MrHonekawa/Batman_Robot#1] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Get Answer as a dev is on
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MrBootsBoots** [comment]
Yes, go
